src/Console/ImportRouteData.php

```diff
@@ -51,28 +53,51 @@ public function __construct()
      */
     public function handle(StopsActivator $stopsActivator)
     {
+        $this->info('Importing route data files...');
         Log::info("NeTEx route data import starting...");
+        $importStatus = ImportStatus::find($this->argument('import-id'));
 
```

Comment:
Post office discussion summary:

- Move all logic around comparing and creating of import status to laravel-netex, (almost) whatever cost.
- Import status is created here in `ImportRoutedata`.
- During activation, latest import status is fetched and updated internally here in laravel-netex.
- Import status is checked wherever needed and handled upon.